packages/fixie/src/main.ts

```diff
@@ -145,13 +145,21 @@ corpus
 const sources = corpus.command('sources').description('Corpus source related commands');
 
 sources
-  .command('add <corpusId> <urlPattern>')
+  .command('add <corpusId> <urls...>')
```

Comment:
Should this be 'urlPatterns' or 'startUrls' or something?

Reply:
Done.